### PR TITLE
New labels map update

### PR DESCRIPTION
**Pull request by frheault**
Matched with
https://github.com/scilus/scilpy/pull/554

Should not change behavior (only minor) on the classic way of using it.

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -243,15 +246,23 @@ process Bundle_Label_And_Distance_Maps {
...
-            --labels_color_dpp ${sid}__\${bname}_labels.trk \
-            --distances_color_dpp ${sid}__\${bname}_distances.trk \
+        scil_compute_bundle_voxel_label_map.py \$bundle \${centroid} tmp_out\
             --min_streamline_count ${params.min_streamline_count} \
-            --min_voxel_count ${params.min_voxel_count}
-        done
+            --min_voxel_count ${params.min_voxel_count} -f --new
+            
+        mv tmp_out/labels_map.nii.gz ${sid}__\${bname}_labels.nii.gz
+        mv tmp_out/corr_map.nii.gz ${sid}__\${bname}_correlation.nii.gz
```
Comment (arnaudbore) [inline main.nf:254]:
```suggestion
        mv tmp_out/correlation_map.nii.gz ${sid}__\${bname}_correlation.nii.gz
```

File: main.nf [hunk 2]
```diff
@@ -243,15 +246,23 @@ process Bundle_Label_And_Distance_Maps {
...
+        mv tmp_out/distances_map.nii.gz ${sid}__\${bname}_distances.nii.gz
+
+        mv tmp_out/_0/mapping_labels.npz ${sid}__\${bname}_labels.npz
+        mv tmp_out/_0/mapping_corr.npz ${sid}__\${bname}_correlation.npz
+        mv tmp_out/_0/mapping_dists.npz ${sid}__\${bname}_distances.npz
+
```
Comment (arnaudbore) [inline main.nf:260]: There is no npz output :|

File: main.nf [hunk 3]
```diff
@@ -243,15 +246,23 @@ process Bundle_Label_And_Distance_Maps {
...
+        mv tmp_out/_0/labels.trk ${sid}__\${bname}_labels.trk
```
Comment (arnaudbore) [inline main.nf:261]:
```suggestion
        mv tmp_out/labels.trk ${sid}__\${bname}_labels.trk
```

File: main.nf [hunk 4]
```diff
@@ -243,15 +246,23 @@ process Bundle_Label_And_Distance_Maps {
...
+        mv tmp_out/_0/corr.trk ${sid}__\${bname}_correlation.trk
```
Comment (arnaudbore) [inline main.nf:262]:
```suggestion
        mv tmp_out/correlation.trk ${sid}__\${bname}_correlation.trk
```

File: main.nf [hunk 5]
```diff
@@ -243,15 +246,23 @@ process Bundle_Label_And_Distance_Maps {
...
+        mv tmp_out/_0/dists.trk ${sid}__\${bname}_distances.trk
```
Comment (arnaudbore) [inline main.nf:263]:
```suggestion
        mv tmp_out/distance.trk ${sid}__\${bname}_distances.trk
```

**Review by arnaudbore — COMMENTED** [review]
Everything seems to fall apart at process : Bundle_Label_And_Distance_Maps

**Comment by frheault** [comment]
@arnaudbore Just tested, working on rc-1.4

**Pull request merged**